mono/mini/interp/interp.c

```diff
@@ -6583,6 +6579,18 @@ interp_exec_method_full (InterpFrame *frame, ThreadContext *context, FrameClause
 
 	g_assert_not_reached ();
 
+abort_common:
+	THROW_EX (mono_get_exception_execution_engine (NULL), ip);
+null:
```

Comment:
`null` is problematic? Anyway I would suggested to prefix every label here with something like `ex_`. Looks good otherwise.